core/types/receipt.go

```diff
@@ -28,10 +28,16 @@ import (
 
 //go:generate gencodec -type Receipt -field-override receiptMarshaling -out gen_receipt_json.go
 
+const (
+	ReceiptStatusSuccess = byte(0x01)
+	ReceiptStatusFailure = byte(0x00)
+)
```

Comment:
These should be private (since we want to work with bools API wise) and please document them.